### DISCUSSION TITLE
next lint isn't showing total warnings and errors

### DISCUSSION THREAD

**Discussion by Crinchy** [vercel/next.js#55170] [Ideas]
### Goals

next lint, should show the total warnings and errors

### Background

Hey, i recently just saw using eslint directly, shows the total amount of warnings and errors, but next lint isnt showing them. Sometimes it is very interesting to see how much warnings or errors there are, while adding new rules to the eslint configuration.

### Proposal

I would love, if you could add the amount of warnings and errors to the next lint command. Thats all ❤

**Comment by atimmer** [comment]
@Crinchy You can achieve this by specifying the `--format` option. I've used ` --format stylish` myself, that's the default ESLint formatter: https://eslint.org/docs/latest/use/formatters/#stylish.

See also https://github.com/vercel/next.js/blob/canary/packages/next/src/cli/next-lint.ts#L92.
